docker-compose.yml

```diff
@@ -39,6 +39,7 @@ services:
             - CELERY_BROKER_URL=redis://redis:6379/0
             - CELERY_RESULT_BACKEND=redis://redis:6379/0
             - BACKEND_INTERNAL_URL=http://yaptide_flask:6000
+            - LOG_LEVEL_CELERY=INFO
```

Comment:
If we set enviroment variable outside docker compose, i.e. in .env file it wont be propagated here

Reply:
Ok, I will add the default value then.

I modified the name of this variable because it actually changes the logging level of `root logger`. 
Not only `celery.*` (tasks etc)

![obraz](https://github.com/yaptide/yaptide/assets/128628266/0be825e0-1b89-46aa-956a-863de8d07536)

Is the new name ok, or maybe the previous one was better?